.github/workflows/test.yml

```diff
@@ -68,6 +70,7 @@ jobs:
       is-web: true
       run: |
         export NODE_OPTIONS="--max-old-space-size=32768"
+        export CHROME_PATH=$HOME/google-chrome-stable/chrome
```

Comment:
Why do we need another workflow instead of running `curl` here?

Reply:
This is a bad usage, I will move it here, it is not ready yet